yt/yt/ytlib/scheduler/config.cpp

```diff
@@ -858,6 +861,11 @@ void TOperationSpecBase::Register(TRegistrar registrar)
             spec->InputTableColumnarStatistics->Enabled = true;
             spec->UserFileColumnarStatistics->Enabled = true;
         }
+
+        if (spec->BatchRowCount) {
+            THROW_ERROR_EXCEPTION_IF(*spec->BatchRowCount <= 0, "Value of \"batch_row_count\" of must be positive");
```

Comment:
This can be expressed via .GreaterThan(0)

Reply:
I think this won't work for std::nullopt? Afair it compares less than any other optional with a value.

Reply:
Indeed!